util/logging.cc

```diff
@@ -52,7 +52,7 @@ bool ConsumeDecimalNumber(Slice* in, uint64_t* val) {
     unsigned char c = (*in)[0];
     if (c >= '0' && c <= '9') {
       ++digits;
-      const int delta = (c - '0');
+      const unsigned int delta = (c - '0');
```

Comment:
This one has been fixed upstream as part of a larger refactor in the new leveldb release.